### PR TITLE
Ers 3062 add slack notification for atom orb fails

### PR DESCRIPTION
**Pull request by hyiu-elementary**
## Change description

Send message to slack channel when circleci steps related to the atom orb fails

## Type of change
- [ ] Bug fix (fixes an issue)
- [X] New feature (adds functionality)

## Related issues

> Fix [#1]() 

### Development

- [x] Lint rules pass locally
- [x] Application changes have been tested thoroughly
- [x] Automated tests covering modified code pass

### Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### Code review 

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached as necessary
- [x] "Ready for review" label attached and reviewers assigned
- [ ] Changes have been reviewed by at least one other contributor
- [ ] Pull request linked to task tracker where applicable

### CODE REVIEW

**Review by nblack-er — APPROVED** [review]

File: .circleci/README.md [hunk 1]
```diff
@@ -125,6 +125,12 @@ circleci orb publish promote elementaryrobotics/atom@dev:some-tag patch
 
 ### Release Notes
 
+#### [v0.3.6](https://circleci.com/orbs/registry/orb/elementaryrobotics/atom?version=0.3.5)
```
Comment (nblack-er) [inline .circleci/README.md:128]: URL needs updating :)

**Comment by hyiu-elementary** [comment]
Peter, Anna, and Kyle, please review and let me know if you have any feedback. This change bumps up the version of the atom orb, and alerts a slack channel when tasks related to the atom orb fails in a circleci pipeline

**Review by aywaldron — APPROVED** [review]
@hyiu-elementary This looks great! Just curious, is there any way to allow configuration of what channel the slack notification goes to, or is it pre-configured somewhere for us?

**Comment by hyiu-elementary** [comment]
By default, all the alerting is configured to go to circleci-pipeline-notifications slack channel. For all the active circleci jobs, I've added an additional environmental variable called "SLACK_DEFAULT_CHANNEL". By adjusting this environment variable for a particular job, we can point it to send to alert to a different slack channel.

**Pull request merged**